.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: ci
+
+on:
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  staticcheck:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v1
```

Comment:
why do we want to use v1 instead of v2?

Reply:
I didn't know it existed haha, i'll update :)